addons/mail/static/src/core/common/record.js

```diff
@@ -1181,8 +1183,8 @@ export class Record {
             const field = fieldOrRecord;
             const rawField = toRaw(field);
             if (type === "compute") {
-                if (!Record.FC_QUEUE.some((f) => toRaw(f) === rawField)) {
-                    Record.FC_QUEUE.push(field);
+                if (!Record.FC_QUEUE.has(rawField)) {
```

Comment:
`add` already check existence internally so the condition is not necessary, I wonder if it would make a difference of if `has` is fast enough.

Reply:
Indeed `has` is useless. Most likely not difference with or without, as `Record.FC_QUEUE` is not wrapped in proxy whatsoever, so this is all about hash lookup of browser of `Set.has` that should be blazing fast so definitely invisible on it being called twice as much as necessary.